lua/telescope/builtin/__lsp.lua

```diff
@@ -179,6 +114,10 @@ local function list_or_jump(action, title, opts)
       vim.list_extend(flattened_results, result)
     end
 
+    if not vim.F.if_nil(opts.results_filter) then
```

Comment:
we should check if `opts.results_filter` is a function with `type`.

Reply:
Good call, done.